### PR TITLE
Removing TODO's

### PR DESCRIPTION
**Pull request by eddynaka**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs [hunk 1]
```diff
@@ -44,16 +44,10 @@ public void FailedToReachCollector(Exception ex)
             }
         }
 
-        [Event(1, Message = "Exporter failed to convert SpanData content into gRPC proto definition. Data will not be sent. Exception: {0}", Level = EventLevel.Error)]
-        public void FailedToConvertToProtoDefinitionError(string ex)
+        [Event(3, Message = "Could not translate activity from class '{0}' and method '{1}', span will not be recorded.", Level = EventLevel.Informational)]
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs:47]: The ordering is not corret. 1,2,3 events in that order would make it easy to maintain.

Comment (eddynaka) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs:47]: the only thing is that i changed from public to private, so people wont use it incorrectly...should i revert it?

File: src/OpenTelemetry/Trace/ActivityProcessorPipelineBuilder.cs [hunk 2]
```diff
@@ -94,8 +94,7 @@ internal ActivityProcessor Build()
             }
             else if (this.Exporter != null)
             {
-                // TODO: Make this BatchingActivityProcessor once its available.
-                exportingProcessor = new SimpleActivityProcessor(this.Exporter);
+                exportingProcessor = new BatchingActivityProcessor(this.Exporter);
```
Comment (cijothomas) [inline src/OpenTelemetry/Trace/ActivityProcessorPipelineBuilder.cs:97]: Don't know what should be the default. Lets keep BatchingProcessor for now. And revisit the area after beta.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs (hunk 1 above)
Comment (eddynaka) [reply src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs:59]: just updated back. i think i did ths with other event sources. i will double-heck

File: src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusExporterEventSource.cs [hunk 3]
```diff
@@ -47,6 +47,18 @@ public void CanceledExport(Exception ex)
             }
         }
 
+        [Event(1, Message = "Failed to export activities: '{0}'", Level = EventLevel.Error)]
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusExporterEventSource.cs:50]: activities -> metrics rename please.

**Pull request merged**